test/integration/monitoring/reschedule-delivery.test.js

```diff
@@ -81,13 +81,13 @@ describe('reschedule deliveries', () => {
 
   test('should send email to requested email address', async () => {
     await rescheduleDelivery(mockDelivery1)
-    expect(mockSendEmail.mock.calls[0][1]).toBe('farmer1@farm.com')
+    expect(mockSendEmail.mock.calls[0][1]).toBe(mockStatement1.email)
   })
 
-  test('should send email with file link', async () => {
-    await rescheduleDelivery(mockDelivery1)
-    expect(mockSendEmail.mock.calls[0][2].personalisation.link_to_file).toBe(MOCK_PREPARED_FILE)
-  })
+  // test('should send email with file link', async () => {
+  //   await rescheduleDelivery(mockDelivery1)
+  //   expect(mockSendEmail.mock.calls[0][2].personalisation.link_to_file).toBe(MOCK_PREPARED_FILE)
+  // })
```

Comment:
Remove if not needed.

Reply:
Good spot -- removed to reduce the output from logs when recording the Notify response object.

Added back in and all passing